src/p5.dance.js

```diff
@@ -1554,7 +1612,43 @@ module.exports = class DanceParty {
       });
     }
 
-    this.p5_.drawSprites();
+    if (!this.externalLayer) {
+      this.p5_.drawSprites();
+    } else {
+      // Draw sprites in two passes, before and after the external layer.
```

Comment:
Out of curiosity, would it be possible to create a regular `Sprite` but [override its `draw` function](https://p5play.org/docs/Sprite#draw) to call the custom renderer at the right time?

Reply:
Great idea. Done.